docs/makeqbk.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
If this works on windows I have no problem. I thought some windows shell had bash in `/usr/bin`.